ads/google/a4a/utils.js

```diff
@@ -205,10 +204,10 @@ export function extractGoogleAdCreativeAndSignature(
             responseHeaders.get(AMP_SIGNATURE_HEADER)));
     }
     if (responseHeaders.has(CREATIVE_SIZE_HEADER)) {
-      const sizeStr = responseHeaders.get(CREATIVE_SIZE_HEADER);
-      // We should trust that the server returns the size information in the
-      // form of a WxH string.
-      size = sizeStr.split('x').map(dim => Number(dim));
+      const sizeArr = responseHeaders.get(CREATIVE_SIZE_HEADER)
+          .split('x')
+          .map(dim => Number(dim));
+      size = {width: sizeArr[0], height: sizeArr[1]};
```

Comment:
let's add a dev().assert for now.  It will get compiled away but its still better than nothing.  We can assume we have e2e coverage to ensure correct behavior (we do have such a thing, correct?)